aws/container-linux/kubernetes/variables.tf

```diff
@@ -59,6 +59,12 @@ variable "disk_type" {
   description = "Type of the EBS volume (e.g. standard, gp2, io1)"
 }
 
+variable "disk_iops" {
+  type        = "string"
+  default     = "120"
+  description = "IOPS of the EBS volume (required for io1)"
```

Comment:
Just reading docs, "0 If the volume is not a provisioned IOPS image, otherwise the supported IOPS count". So if we choose to permit it, this shouldn't default to 120.